doxygen/lang/900_release_notes.dox.tmpl

```diff
@@ -7,6 +7,14 @@
     @par Release Summary
     Bugfix release; see details below
 
+    @subsection qore_08137_new_features New Features in Qore
+    - improved debugging support:
+        - @ref Qore::ProgramControl::getStatementIdInfo() "ProgramControl::getStatementIdInfo()" provides breakpoint info
+        - command to resolve statement
+        - interrupt notification provides call stack info
+        - "Run to statement" implementation
+        - @ref DebugHandler reimplemented to support multiple websocket handlers
```

Comment:
`@ref DebugHandler` will not work; you have to use `<a href="../../modules/DebugHandler/html/index.html">DebugHandler</a>` instead.  Please always generate the documentation and review the output before submitting a pull request